src/Lib/Security/Hash.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types = 1);
+namespace App\Lib\Security; // pak vyuziti bude use App\Lib\Security\Hash;
+
+// use App\Lib\Configuration\ConfigurationFactory;
+//$configuration = ConfigurationFactory::fromFileName('common');
+//$salt = $configuration->get('jwt_key'); // change to salt_key?
+
+class Hash
+{
+
+
+  /**
+  * Creates hash from string.
+  *
+  * @return string
+  */
+    public static function getHash(string $str):string
+    {
+        $salt = 'hellomysalt';
+
+        $res = hash('whirlpool', $salt.$str);
```

Comment:
Did you try password_hash? https://www.php.net/manual/en/function.password-hash.php with Argon2id algo it will automaticli add salt. This is good article about crypto https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#hashing-concepts